lib/utils/isStandardSyntaxSelector.js

```diff
@@ -12,7 +12,7 @@ module.exports = function(selector /*: string*/) /*: boolean*/ {
   }
 
   // SCSS placeholder selectors
-  if (selector.indexOf("%") === 0) {
+  if (selector[0] === "%") {
```

Comment:
Regression, i can use `.my-class%foo { }`

Reply:
```js
".my-class%foo".indexOf("%") === 0
// false
```

Reply:
`selector.startsWith("%")` is _much_ clearer in the intent... assuming that I'm reading the intent of that code correctly in the first place!

Reply:
fixed in https://github.com/stylelint/stylelint/pull/3405/commits/48d4788f3a6559fb4188b1d1efb4824ced319afe